captum/insights/api.py

```diff
@@ -339,20 +346,21 @@ def _calculate_vis_output(
         scores = scores.cpu().squeeze(0)
         predicted = predicted.cpu().squeeze(0)
 
+        actual_label_output = None
         if label is not None and len(label) > 0:
             actual_label_output = OutputScore(
                 score=100, index=label[0], label=self.classes[label[0]]
             )
-        else:
-            actual_label_output = None
 
         predicted_scores = self._get_labels_from_scores(scores, predicted)
 
         # Filter based on UI configuration
-        if not self._should_keep_prediction(predicted_scores, actual_label_output):
+        if actual_label_output is None or not self._should_keep_prediction(
```

Comment:
nit: Is this preferable over keeping this line the same and just changing the line in the function to return False if actual_label_output is None?

Reply:
Typechecker isn't happy later on when returning the `VisualizationOutput`